protected/views/layouts/datasetpage.php

```diff
@@ -48,6 +48,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.ba-bbq/1.2.1/jquery.ba-bbq.min.js" ></script>
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.0.4/css/bootstrap.min.css">
```

Comment:
datasetpage.php is the layout used by most of the inner pages of the admin section

2.0.4 is the same Bootstrap version mentioned in the local bootstrap files

Using a +3 bootstrap version breaks the styles significantly, so if they have to be revamped anyway, I think it's best to handle the revamp directly, rather than fixing the current ones